src/sync.js

```diff
@@ -118,13 +110,13 @@ Sync.prototype = {
 
   queueGetRequest: function (path) {
     return new Promise((resolve, reject) => {
-      if (!this.remote.connected) {
+      if (!this.rs.remote.connected) {
```

Comment:
Instead of all the changes to `this.rs.remote` and `this.rs.local` everywhere, I would rather set a local reference (`this.remote = this.rs.remote`) and keep accessing it via `this.remote`.

Reply:
That's exactly how it worked before (except for unnecessary constructor arguments). If you read my commit description, then you'll find that this is intentional in order to make it clear that this is a remoteStorage instance's remote, and also make it easier to rip that out in the future, because why should there be `remoteStorage.sync.remoteStorage` in the first place.